layers/meta-balena-allwinner/recipes-bsp/u-boot/u-boot_2017.11/0003-libftd_fix.patch

```diff
@@ -0,0 +1,35 @@
+From 6eecc74a9178ddd29375a6c3ee354f2e8b989c45 Mon Sep 17 00:00:00 2001
+From: Vicentiu Galanopulo <vicentiu@balena.io>
+Date: Thu, 26 Sep 2019 16:19:02 +0200
+Subject: [PATCH] Include U-Boot libfdt headers from their actual path
+
+There are no headers for libfdt in lib/libfdt, as they are instead
+located in scripts/dtc/libfdt. Specifying lib/libfdt for headers
+inclusion in host tools results in using the system libfdt headers,
+which is not what we want. Change this to the proper path.
+
+Signed-off-by:Paul Kocialkowski <contact@paulk.fr>
+
+Upstream-Status: Inappropriate [Not author]
```

Comment:
Is this already in upstream? If yes, then status should be instead `Backport`

Reply:
Done